### PR TITLE
Feat: Added updates section [#1450]

### PR DESCRIPTION
**Pull request by Abhishek9503**
## Description

#1450
In this PR i have added the the update section with all the filter functionality

## What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build

## Have you tested locally?

- [x] 👍 yes
- [ ] 🙅 no, because I am lazy

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md

### CODE REVIEW

**Review by Astitva877 — APPROVED** [review]

File: src/packages/@deprecate/utils/constants.ts [hunk 1]
```diff
@@ -11,6 +11,7 @@ const constants = {
   SPARROW_SUPPORT_EMAIL: import.meta.env.VITE_SPARROW_SUPPORT_EMAIL,
   SPARROW_AUTH_URL: import.meta.env.VITE_AUTH_URL,
   SPARROW_GITHUB: import.meta.env.VITE_SPARROW_GITHUB,
+  SPARROW_LINKEDIN: import.meta.env.VITE_SPARROW_LINKEDIN,
   SPARROW_DOWNLOAD_LINK: import.meta.env.VITE_SPARROW_DOWNLOAD_LINK,
```
Comment (itsmdasifraza) [inline src/packages/@deprecate/utils/constants.ts:15]: add this env to example env file

File: src/packages/@library/icons/LinkedinIcon.svelte [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<script>
+    export let height = "24px";
+    export let width = "24px";
+    export let color = "grey";
+  </script>
+  
+
+<svg {width}
+{height} viewBox="0 0 18 18" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M12 6C13.1935 6 14.3381 6.47411 15.182 7.31802C16.0259 8.16193 16.5 9.30653 16.5 10.5V15.75H13.5V10.5C13.5 10.1022 13.342 9.72064 13.0607 9.43934C12.7794 9.15804 12.3978 9 12 9C11.6022 9 11.2206 9.15804 10.9393 9.43934C10.658 9.72064 10.5 10.1022 10.5 10.5V15.75H7.5V10.5C7.5 9.30653 7.97411 8.16193 8.81802 7.31802C9.66193 6.47411 10.8065 6 12 6Z" stroke="#808080" stroke-linecap="round" stroke-linejoin="round"/>
```
Comment (itsmdasifraza) [inline src/packages/@library/icons/LinkedinIcon.svelte:10]: color is not used

File: src/packages/@support/features/release-notes/layout/ReleaseNotes.svelte [hunk 3]
```diff
@@ -367,7 +431,7 @@
     left: -140px;
     top: 10px;
     font-weight: bold;
-    color: #ffffff;
+    color: var(--white-color);
     width: 120px;
     text-align: right;
     font-weight: 700;
```
Comment (itsmdasifraza) [inline src/packages/@support/features/release-notes/layout/ReleaseNotes.svelte:437]: use upgraded color code

**Review by itsmdasifraza — APPROVED** [review]

**Pull request merged**